ci/azure/windows.yml

```diff
@@ -17,7 +17,9 @@ jobs:
         CONDA_PY: "37"
 
   steps:
-    - powershell: Write-Host "##vso[task.prependpath]$env:CONDA\Scripts"
+    - powershell: |
+        Write-Host "##vso[task.prependpath]$env:CONDA\Scripts"
+        Write-Host "##vso[task.prependpath]$HOME/miniconda3/bin"
```

Comment:
@bhavaniravi  any idea if there is a way to do this without powershell?  now that windows supports bash, im trying to see if we can get rid of windows-specific stuff

Reply:
In bash this is a simple echo I think.

Reply:
Thats what I thought too, see #30393 where I tried that in the first commit and reverted it when that failed